src/Particles/babylon.solidParticleSystem.ts

```diff
@@ -0,0 +1,427 @@
+            this._rotMatrix.m[2] = 2.0 * (this._quaternion.z * this._quaternion.x - this._quaternion.y * this._quaternion.w);
+            this._rotMatrix.m[3] = 0;
+            this._rotMatrix.m[4] = 2.0 * (this._quaternion.x * this._quaternion.y - this._quaternion.z * this._quaternion.w);
+            this._rotMatrix.m[5] = 1.0 - (2.0 * (this._quaternion.z * this._quaternion.z + this._quaternion.x * this._quaternion.x));
+            this._rotMatrix.m[6] = 2.0 * (this._quaternion.y * this._quaternion.z + this._quaternion.x * this._quaternion.w);
+            this._rotMatrix.m[7] = 0;
+            this._rotMatrix.m[8] = 2.0 * (this._quaternion.z * this._quaternion.x + this._quaternion.y * this._quaternion.w);
+            this._rotMatrix.m[9] = 2.0 * (this._quaternion.y * this._quaternion.z - this._quaternion.x * this._quaternion.w);
+            this._rotMatrix.m[10] = 1.0 - (2.0 * (this._quaternion.y * this._quaternion.y + this._quaternion.x * this._quaternion.x));
+            this._rotMatrix.m[11] = 0;
+            this._rotMatrix.m[12] = 0;
+            this._rotMatrix.m[13] = 0;
+            this._rotMatrix.m[14] = 0;
+            this._rotMatrix.m[15] = 1.0
+        }
+
+        // dispose the SPS
+        public dispose(): void {
+            this.mesh.dispose();
+            this.mesh = null;
+            this.particles.length = 0;
+            this.nbParticles = null;
+            this.billboard = null;
+            this.counter = null;
+            this._scene = null;
+            this._positions.length = 0;
+            this._indices.length = 0;
+            this._normals.length = 0;
+            this._colors.length = 0;
+            this._uvs.length = 0;
+            this._index = null;
+            this._shapeCounter = null;
+            this._useParticleColor = null;
+            this._useParticleTexture = null;
+            this._useParticleRotation = null;
+            this._useParticleVertex = null;
+            this._cam_axisZ = null;
+            this._cam_axisY = null;
+            this._cam_axisX = null;
+            this._axisX = null;
+            this._axisY = null;
+            this._axisZ = null;
+            this._camera = null;
+            this._fakeCamPos = null;
+            this._rotMatrix = null;
+            this._invertedMatrix = null;
+            this._rotated = null;
+            this._quaternion = null;
+            this._vertex = null;
+            this._yaw = null;
+            this._pitch = null;
+            this._roll = null;
+            this._halfroll = null;
+            this._halfpitch = null;
+            this._halfyaw = null;
+            this._sinRoll = null;
+            this._cosRoll = null;
+            this._sinPitch = null;
+            this._cosPitch = null;
+            this._sinYaw = null;
+            this._cosYaw = null;   
+        }
+
+        // Optimizers
+        public enableParticleRotation(): void {
```

Comment:
mmh... I don't really get what I should do
do you prefer  : 

```javascript
public static set ParticleRotation(rot) {
  this._useParticleRotation = rot;
}
```
instead of enable/disable function ?